src/main/java/com/ericsson/ei/controller/model/ResponseEntity.java

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
Same comment here,

Reply:
This is not possible since these are generated based on RAML Schemas. No file in the model package has the header.

Reply:
ok